src/main/java/com/serenitydojo/calculator/Calculator.java

```diff
@@ -1,7 +1,90 @@
 package com.serenitydojo.calculator;
 
+import static org.apache.commons.lang3.StringUtils.isNumeric;
+
 public class Calculator {
+    private String operator = "";
+    private int evaluation = 0;
```

Comment:
Minor point - making the evaluation variable a field means that the class will not be thread-safe - if two threads called the same object in parallel, the calculation would be wrong. It's not really relevant for this scenario, but worth knowing.